scala/build/Dockerfile.maintenance

```
@@ -0,0 +1,259 @@
+
+# sbt build
+RUN sbt sbtVersion
+
+# The scala server will run on port 9000 by default
+EXPOSE 9000
+
+# nginx port
+EXPOSE 8080 
+
+# Install mocha for API testing
+RUN npm install -g mocha
```

Hadolint

[warning] Pin versions in npm. Instead of `npm install <package>` use `npm install <package>@<version>`

(DL3016)